lib/srv/db/cloud/watchers/watcher.go

```diff
@@ -22,11 +22,12 @@ import (
 
 	"github.com/gravitational/trace"
 	"github.com/sirupsen/logrus"
-	"golang.org/x/exp/slices"
 
 	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/teleport/lib/cloud"
 	"github.com/gravitational/teleport/lib/services"
+	discovery "github.com/gravitational/teleport/lib/srv/discovery/common"
+	"github.com/gravitational/teleport/lib/srv/discovery/fetchers/db"
```

Comment:
I will move `lib/srv/cloud/watchers/watchers.go` and its test to `lib/srv/cloud` as they are the only files left in this folder. Currently here for the nicer diff but will move before merge into master.